lib/composer/wikia/fluent-sql-php/src/SQL.php

```diff
@@ -871,14 +878,14 @@ public function HAVING($column) {
 	 * @param $limit
 	 * @return SQL
 	 */
-	public function LIMIT($limit) {
-		$this->limit = new Clause\Limit($limit);
+	public function LIMIT($limit, $allowNonNumeric = false) {
```

Comment:
Should we enable non numeric values in limit at all? Is it used anywhere as such?